project/management/commands/fixnewlines.py

```diff
@@ -19,7 +27,12 @@ def iter_get_repo_files(repo_root: Path=BASE_DIR) -> Iterator[Path]:
     for pathname in pathnames:
         # git ls-tree always outputs posix-style pathnames, even on Windows.
         parts = pathname.split('/')
-        yield repo_root.joinpath(*parts)
+        path = repo_root.joinpath(*parts)
+        # It's possible the path has been removed in the current branch
+        # but hasn't been committed yet, so test to see if it exists
+        # first.
+        if path.exists():
+            yield path
```

Comment:
Ugh, ran into this when moving files around in this PR, see the comments for more details.